radio/src/Makefile

```diff
@@ -52,7 +52,7 @@ DELAY = 5
 # PCB version
 # Values: 9X, 9X128, 9X2561, 9XR, 9XR128, 9XR2561, GRUVIN9X, MEGA2560, SKY9X, 9XRPRO, AR9X, TARANIS
 # Old values STD and STD128 are mapped to 9X and 9X128 for compatibility
-PCB = 9X
+PCB = 9XRPRO
```

Comment:
On 11/12/15 18:46, Andre Bernet wrote:
>
> It would have worked that way if you had created your branch /before/
> commtting your makefile changes on master (or alternatively rebased it
> to a point of master before these changes were committed after the fact)

Definitely. That was not what I wanted though, I wanted to make the 
changes to the makefile permanent in my own branch, so that I could 
build with my settings.

Having the makefile under revision control is a bit problematic anyway, 
or at least it would be nice for people building from source to have a 
config file where they could store their build settings that did not 
interfere with version control.

(I have not done much with git in the past, so I will first have to get 
a grip on git-ish features like rebase.)


-- 
GCS$/CC/E/IT d- s+ a C++ UL+++/S++ P L++ E-
W++ N o? K? w O(++) M-- V+ PS+ PE+ Y? PGP+ t
5? X R tv b++ DI++ D++ G e+++ h---- r+++ y+++